src/sb_utils/test/rviz-utils-rostest.cpp

```diff
@@ -0,0 +1,276 @@
+/*
```

Comment:
Just for naming convention's sake, this should be `rviz_utils_rostest.cpp`

Reply:
!fixed